setup.py

```diff
@@ -110,13 +115,14 @@ def build_extension(self, ext):
             cmake_args += ['-DCMAKE_BUILD_TYPE=' + cfg]
             build_args += ['--', '-j2']
 
-        env = os.environ.copy()
+        print(f"PATH={env['PATH']}")
```

Comment:
Remove various print statements I assume used for debugging

Reply:
Thanks for catching this! will do!